packages/grafana-data/src/transformations/transformers/ids.ts

```diff
@@ -3,6 +3,8 @@ export enum DataTransformerID {
   append = 'append', // Merge all series together
   //  rotate = 'rotate', // Columns to rows
   reduce = 'reduce', // Run calculations on fields
+  sort = 'sort', // sort fields based on user configuration
+  sortAndFilter = 'sortAndFilter', // sort and filter based on user configuration
```

Comment:
Rather than "sort" maybe "order" or "reorder"?

With sort I assumed it would be A-Z etc

Reply:
OK ok, great feedback!

Reply:
also... not sure we need *two* transformers, lets start with one that does everything and can have empty options:
* reorder
* filter
* rename

Not sure what to call the transformer though... maybe "organize" ;)

Reply:
Yeah, I like organize :) OrganizeFieldsTransformer.

hmm, I don't think I agree regarding the number of transformers. I would prefer to keep this transformer as a combination of smaller simpler transformers. It makes it easier to test and understand. It also makes it possible to reuse them one and one. If that makes sense?